contracts/sg721-updatable/src/contract.rs

```diff
@@ -0,0 +1,367 @@
+use semver::Version;
+
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    to_binary, Binary, Decimal, Deps, DepsMut, Empty, Env, Event, MessageInfo, StdError, StdResult,
+    Uint128,
+};
+use cw2::set_contract_version;
+
+use cw_utils::nonpayable;
+use sg_std::fees::burn_and_distribute_fee;
+use sg_std::StargazeMsgWrapper;
+
+use crate::ContractError;
+use cw721::ContractInfoResponse;
+use cw721_base::ContractError as BaseError;
+use url::Url;
+
+use crate::msg::{
+    CollectionInfoResponse, ExecuteMsg, InstantiateMsg, QueryMsg, RoyaltyInfoResponse,
+};
+use crate::state::{CollectionInfo, RoyaltyInfo, COLLECTION_INFO, FROZEN_TOKEN_METADATA};
+
+// version info for migration info
+const COMPATIBLE_MIGRATION_CONTRACT_NAME: &str = "crates.io:sg-721";
+const CONTRACT_NAME: &str = "crates.io:sg721-updatable";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+const CREATION_FEE: u128 = 1_000_000_000;
+
+// Disable instantiation for production build
```

Comment:
Don't need this anymore?

Reply:
we need it I disabled because I though it was something wrong with my tests but should be added back